airflow/config_templates/default_airflow.cfg

```diff
@@ -111,6 +111,14 @@ sql_alchemy_connect_timeout_seconds = 0
 # for postgres is usually specified in milliseconds, but this uses seconds.
 sql_alchemy_statement_timeout_seconds = 0
 
+# Maximum number of attempts to perform statement on the database. Set this to <= 1
+# to disable retries and perform a single attempt.
+sql_alchemy_statement_max_tries = 1
```

Comment:
nit: It makes much more sense to me to talk about this as `num_retries` instead of `num_tries`, as that's how it's normally done

Reply:
Sure we can make this `retries` everywhere if that's easier to reason about.